templates/slides/projector_countdown.html

```diff
@@ -1,6 +1,6 @@
 <link rel="stylesheet" type="text/css" href="/system/projector/static/slide/projector_countdown.css" />
-
-<div class="countdown {{ if .Options.Fullscreen }}fullscreen{{ else }}overlay{{ end }}">
+<!-- prettier-ignore -->
+<div class="countdown {{ if .Options.Fullscreen }}fullscreen{{ else }}overlay{{ end }}" data-is-overlay="{{ if not .Options.Fullscreen }}true{{ else }}false{{ end }}">
```

Comment:
You could just drop the attribute if the condition is not fulfilled instead of setting it to false